site/resources.md

```diff
@@ -8,7 +8,7 @@ Here you will find external resources about Velero, such as videos, podcasts, an
 
 ## Previous community meetings
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe title="Velero Community Meeting - Oct 22, 2019" width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PL7bmigfV0EqQRysvqvqOtRNk4L5S7uqwM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
That title should be "Velero Community Meetings/Open Discussions", as it's a playlist, not a single video.